shared/src/native-src/environment_variables.h

```diff
@@ -17,6 +17,23 @@ namespace shared {
 		// "/var/log/datadog/dotnet/" on Linux.
 		const WSTRING log_directory = WStr("DD_TRACE_LOG_DIRECTORY");
 
+
+		//
+		// LOADER FEATURE FLAGS
+		//
+
+		// Disables rewriting the module initializer
+		const WSTRING disable_rewrite_module_initializer = WStr("DD_LOADER_DISABLE_REWRITE_MODULE_INITIALIZER");
+
+		// Disables rewriting the module entrypoint
+		const WSTRING disable_rewrite_module_entrypoint = WStr("DD_LOADER_DISABLE_REWRITE_MODULE_ENTRYPOINT");
+
+		// Disables rewriting the mscorlib
+		const WSTRING disable_rewrite_mscorlib = WStr("DD_LOADER_DISABLE_REWRITE_MSCORLIB");
+
+		// Disables NGEN images support
+		const WSTRING disable_ngen = WStr("DD_LOADER_DISABLE_NGEN");
```

Comment:
We usually use `DD_{feature}_ENABLED` for feature flags:
```
DD_LOADER_REWRITE_MODULE_INITIALIZER_ENABLED
DD_LOADER_REWRITE_MODULE_ENTRYPOINT_ENABLED
DD_LOADER_REWRITE_MSCORLIB_ENABLED
DD_LOADER_NGEN_ENABLED
```

Of course, if you change them you will also need to reverse all your logic.

Reply:
They would also match the elements in `LoaderOptions` (`RewriteModulesInitializers`, `RewriteMSCorLibMethods`, etc).

Reply:
You're right!, I agree, I made the change in 742ab68